bears/general/LineCountBear.py

```diff
@@ -10,14 +12,24 @@ class LineCountBear(LocalBear):
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting'}
 
-    def run(self, filename, file, max_lines_per_file: int):
+    def get_blank_line_count(self, file):
+        num_blank_lines = len(
+            list(filter(lambda x: re.match(r'^\s*$', x), file)))
+        return num_blank_lines
+
+    def run(self, filename, file, max_lines_per_file: int,
+            exclude_blank_lines: bool=False):
         """
         Count the number of lines in a file and ensure that they are
         smaller than a given size.
 
         :param max_lines_per_file: Number of lines allowed per file.
+        :param exclude_blank_lines: "True" if blank lines are to be excluded.
         """
         file_length = len(file)
+        if exclude_blank_lines:
+            num_blank_lines = LineCountBear.get_blank_line_count(self, file)
```

Comment:
`get_blank_line_count` should be a static method as it is not using the bear instance